### PR TITLE
[iOS][Android] ActiveIssue SpanTests and Matrix4x4Tests that are failing on CI

### PR DESCRIPTION
**Pull request by steveisok**
[Matrix4x4Tests.Matrix4x4IndexerSetTest test fails](https://github.com/dotnet/runtime/issues/80876)

[SpanTests.IndexOfAnyValues_GetValues test fails trying to JIT in AOT mode](https://github.com/dotnet/runtime/issues/80875)

### CODE REVIEW

File: src/libraries/System.Reflection.Metadata/tests/Metadata/MetadataReaderTests.cs [hunk 1]
```diff
@@ -3072,6 +3072,7 @@ private static unsafe byte[] ObfuscateWithExtraData(byte[] unobfuscated, bool se
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/80863", TestPlatforms.Android | TestPlatforms.iOS | TestPlatforms.tvOS)]
```
Comment (lewing) [inline src/libraries/System.Reflection.Metadata/tests/Metadata/MetadataReaderTests.cs:3075]: https://github.com/dotnet/runtime/pull/80878

**Review by lewing — APPROVED** [review]
Changes look ok, might not need the last one long term

**Review by simonrozsival — APPROVED** [review]

**Pull request merged**